### PR TITLE
Added nanoFramework.OpenThread library

### PR DESCRIPTION
**Pull request by vgolovanov**
## Checklist:

- [x] I've placed my contribution in the appropriate section.
- [x] I've choosen a good name for it and I've checked the repo for duplicates.
- [x] I've added a readme.md following the provided template (TBD). 
- [x] My code follows the code style of this project.
- [x] I've tidy my files inside the contribution folder.
- [x] I've edited the repo README.md to include the mention and link to my contribution. It's under the proper section and the link is correct.
- [x] The code builds.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -55,8 +55,11 @@ These contributions are **NOT** supported by the **nanoFramework** team and any
   <td><a href="libraries/LoRa-RFM9X">Library for controlling a LoRa HopeRF RFM9X</a></td>
   <td><a href="libraries/LoRa-RFM9X">Library for controlling a LoRa RAK811</a></td> 
   <td><a href="libraries/ThingsBoardDotNet">Library to work with ThingsBoard IoT Platform</a></td> 
-  <td><a href="libraries/nanoFramework.CoAP">nanoFramework.CoAP library</a></td>
-  <td><!--<a href="Utility/util3">Utility Three</a>--></td>
+  <td><a href="libraries/nanoFramework.CoAP">nanoFramework.CoAP library</a></td>  
+ </tr>
```
Comment (Ellerbach) [inline README.md:59]: Can you align properly the table markers? and create enough for the table to be correctly aligned?

**Review by Ellerbach — APPROVED** [review]
Thanks for the contribution! Just a little element to adjust

File: README.md (hunk 1 above)
Comment (vgolovanov) [reply README.md:59]: HTML table fixed.

File: README.md [hunk 2]
```diff
@@ -55,8 +55,13 @@ These contributions are **NOT** supported by the **nanoFramework** team and any
...
```
Comment (Ellerbach) [inline README.md:59]: I know you'll call me picky but can you please also properly align the spaces for the tables.
Thanks a lot.

Comment (vgolovanov) [reply README.md:59]: Tabs vs Spaces :-) I just copy and pasted additional rows and didn't pay attention to formatting. :-) Fixed.

**Review by josesimoes — APPROVED** [review]

**Pull request merged**